Screens/Programs.js

```diff
@@ -198,7 +212,7 @@ const styles = StyleSheet.create({
   middleContainer: {
     flexDirection: "column",
 
-    height: 500,
+    height: "70%",
```

Comment:
Nice! We should really be changing everything to percentages. Good initiative :)